src/App.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import Note from './components/Note'
+import { useStyles } from "./styles/styles";
+
+function App (props) {
+    const classes = useStyles(props);
```

Comment:
Remove please `useStyles()` from component. Try to get styles by using `styled components` or `make` function.